asistente_ladm_col/resources/toml/LADM_COL_configuration.toml

```diff
@@ -51,3 +53,464 @@ form_order=8
 
 ["LADM_COL_V3_0.LADM_Nucleo.ExtDireccion.Valor_Via_Principal"]
 form_order=5
+
+# LC_Predio
+
+["LADM_COL_V3_0.LADM_Nucleo.COL_UnidadAdministrativaBasica.Nombre"]
+form_order=13
+
+["LADM_COL_V3_0.LADM_Nucleo.ObjetoVersionado.Comienzo_Vida_Util_Version"]
+form_order=14
+
+["LADM_COL_V3_0.LADM_Nucleo.ObjetoVersionado.Fin_Vida_Util_Version"]
+form_order=15
+
+["LADM_COL_V3_0.LADM_Nucleo.Oid.Espacio_De_Nombres"]
+form_order=16
+
+["LADM_COL_V3_0.LADM_Nucleo.Oid.Local_Id"]
+form_order=17
+
+["Modelo_Aplicacion_LADMCOL_Lev_Cat_V1_0.Levantamiento_Catastral.LC_Predio.Avaluo_Catastral"]
+form_order=12
+
+["Modelo_Aplicacion_LADMCOL_Lev_Cat_V1_0.Levantamiento_Catastral.LC_Predio.Codigo_ORIP"]
+form_order=4
+
+["Modelo_Aplicacion_LADMCOL_Lev_Cat_V1_0.Levantamiento_Catastral.LC_Predio.Condicion_Predio"]
+form_order=11
+
+["Modelo_Aplicacion_LADMCOL_Lev_Cat_V1_0.Levantamiento_Catastral.LC_Predio.Departamento"]
+form_order=1
+
+["Modelo_Aplicacion_LADMCOL_Lev_Cat_V1_0.Levantamiento_Catastral.LC_Predio.Direccion"]
+form_order=9
+
+["Modelo_Aplicacion_LADMCOL_Lev_Cat_V1_0.Levantamiento_Catastral.LC_Predio.Id_Operacion"]
+form_order
```

Comment:
You need to set an order number for this field!

Reply:
thanks